src/services/MobileService.ts

```diff
@@ -282,12 +276,16 @@ class MobileService {
     try {
       const exchangeUserId = Number(req.params['exchangeUserId'])
       const backupMethod = req.query.backupMethod || 'UNKNOWN'
+      const backupSharePairId = req.query.backupSharePairId
```

Comment:
Will this get added from the SDK? Are current implementations of multi-backup require the custodian to be reading the backupSharePairId from the query param?

Reply:
Blake is using it for v3 multi-wallet